src/layouts/Desktop/components/NavBar/NavBar.tsx

```diff
@@ -44,7 +40,7 @@ const NavBar = () => {
           </div>
         </div>
         <div className='navbar-contain-user'>
-          <NavBarUser user={users} />
+          <NavBarUser user={user} />
```

Comment:
if NavBarUser is presentational component it is a good solution, but if NavBarUser has some logic inside, you could simply move hook inside of it